### PR TITLE
[UG] Add new section on generating of reports

### PR DESCRIPTION
**Pull request by Eclmist**
Adds guide to generating statistics report:
image

Currency notation in `setrate` has also been changed from 40$ -> $40.00

`Report` command has also been added to the command summary section:
image

### CODE REVIEW

**Review by PotatoCombat — DISMISSED** [review]
LGTM! Thanks!

**Review by PotatoCombat — COMMENTED** [review]
If the rate has been changed to allow cents, then the command needs to be updated to pe a double instead of an integer

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -1065,7 +1066,37 @@ Format: `setrate RATE`
...
+==== Generate Statistic Report : `report`
+
+You can use this command to generate a report to display information such as:
+
+* A breakdown and summary of completed sessions
+* The number of hours of each type of completed sessions
+* A breakdown of your student's ratings
+
+Optionally, you can specify a module code. If a module code is specified, the report generated will only include data from the specified module.
+
```
Comment (fatin99) [inline docs/UserGuide.adoc:1084]: Could you help me add a line below this to say that pressing ESC closes the window? thanks

Comment (Eclmist) [reply docs/UserGuide.adoc:1084]: Alright

**Review by aakanksha-rai — APPROVED** [review]
LGTM

**Pull request merged**